nixos/modules/programs/steam.nix

```diff
@@ -68,6 +71,23 @@ in {
       '';
     };
 
+    extraCompatPackages = mkOption {
+      type = with types; listOf package;
+      default = [ ];
+      defaultText = literalExpression "[]";
+      example = literalExpression ''
+        with pkgs; [
+          luxtorpeda
+          proton-ge
```

Comment:
Are these two in Nixpkgs? If not, remove them or add a note about that and where to get these.

Reply:
as written in the pr description, there is no proton derivation in nixpkgs atm, so i removed that.